components/application-broker/internal/broker/testing/client_assertion.go

```diff
@@ -5,22 +5,23 @@ import (
 
 	"github.com/google/go-cmp/cmp"
 
+	istioversionedclientfake "istio.io/client-go/pkg/clientset/versioned/fake"
```

Comment:
(nit): I see you used fakeistioclientset([here](components/application-broker/internal/broker/testing/listers.go)) which is great IMO. Lets be consistent.